appmonitoring/ts/src/Constants.ts

```diff
@@ -0,0 +1,4 @@
+export const WebhookDNSEndpoint = 'app-monitoring-webhook-service.kube-system.svc'; //This is also defined in _omsagent.yaml in AKS RP and must always match the value there
```

Comment:
Let's make it clear that the comment is for the entire code block, not just the first line. They way you did it - it seems to only relate to the first line.

Reply:
ah yeah good point